### PR TITLE
QuickReply with TemplateMessageBuilder

### PR DESCRIPTION
**Pull request by vocolboy**
#165

### CODE REVIEW

File: src/LINEBot/Constant/ActionType.php [hunk 1]
```diff
@@ -24,4 +24,7 @@ class ActionType
     const POSTBACK = 'postback';
     const URI = 'uri';
     const DATETIME_PICKER = 'datetimepicker';
+    const CAMERA = 'camera';
```
Comment (Vaduz) [inline src/LINEBot/Constant/ActionType.php]: Can I add example message to [TextMessageHandler.php](https://github.com/line/line-bot-sdk-php/blob/master/examples/KitchenSink/src/LINEBot/KitchenSink/EventHandler/MessageHandler/TextMessageHandler.php) using these new action types?

Comment (vocolboy) [reply src/LINEBot/Constant/ActionType.php]: Suchlike this one?
```
case 'carousel':
    $imageUrl = UrlBuilder::buildUrl($this->req, ['static', 'buttons', '1040.jpg']);
    $quickReply = new QuickReplyMessageBuilder([
        new QuickReplyButtonBuilder(new LocationTemplateActionBuilder('Location')),
    ]);
    $carouselTemplateBuilder = new CarouselTemplateBuilder([
        new CarouselColumnTemplateBuilder('foo', 'bar', $imageUrl, [
            new UriTemplateActionBuilder('Go to line.me', 'https://line.me'),
            new PostbackTemplateActionBuilder('Buy', 'action=buy&itemid=123'),
        ]),
        new CarouselColumnTemplateBuilder('buz', 'qux', $imageUrl, [
            new PostbackTemplateActionBuilder('Add to cart', 'action=add&itemid=123'),
            new MessageTemplateActionBuilder('Say message', 'hello hello'),
            ]),
        ]);
    $templateMessage = new TemplateMessageBuilder('Button alt text', $carouselTemplateBuilder,$quickReply);
    $this->bot->replyMessage($replyToken, $templateMessage);
```

**Comment by vocolboy** [comment]
@Vaduz Yes, I already signed CCLA. 
Thanks for your friendly reminder.

File: src/LINEBot/Constant/ActionType.php (hunk 1 above)
Comment (Vaduz) [reply src/LINEBot/Constant/ActionType.php]: Users can see quickReply message when user send message like `quickReply` would be helpful.
The sending message is good as same as the message you wrote in the test case.

Comment (vocolboy) [reply src/LINEBot/Constant/ActionType.php]: @Vaduz Okay, Would you like me to add example message to TextMessageHandler.php ?

Comment (Vaduz) [reply src/LINEBot/Constant/ActionType.php]: Like this:

```PHP
use LINE\LINEBot\QuickReplyBuilder\QuickReplyMessageBuilder;
use LINE\LINEBot\QuickReplyBuilder\ButtonBuilder\QuickReplyButtonBuilder;
use LINE\LINEBot\TemplateActionBuilder\CameraRollTemplateActionBuilder;
use LINE\LINEBot\TemplateActionBuilder\CameraTemplateActionBuilder;
use LINE\LINEBot\TemplateActionBuilder\DatetimePickerTemplateActionBuilder;
use LINE\LINEBot\TemplateActionBuilder\LocationTemplateActionBuilder;
use LINE\LINEBot\MessageBuilder\TextMessageBuilder;
...
(snip)
...
            case 'quickReply':
                $postback = new PostbackTemplateActionBuilder('Buy', 'action=quickBuy&itemid=222', 'Buy');
                $datetimePicker = new DatetimePickerTemplateActionBuilder(
                    'Select date',
                    'storeId=12345',
                    'datetime',
                    '2017-12-25t00:00',
                    '2018-01-24t23:59',
                    '2017-12-25t00:00'
                );

                $quickReply = new QuickReplyMessageBuilder([
                    new QuickReplyButtonBuilder(new LocationTemplateActionBuilder('Location')),
                    new QuickReplyButtonBuilder(new CameraTemplateActionBuilder('Camera')),
                    new QuickReplyButtonBuilder(new CameraRollTemplateActionBuilder('Camera roll')),
                    new QuickReplyButtonBuilder($postback),
                    new QuickReplyButtonBuilder($datetimePicker),
                ]);
                
                $messageTemplate = new TextMessageBuilder('Text with quickReply buttons', $quickReply);
                $this->bot->replyMessage($replyToken, $messageTemplate);
                break;
```

Comment (vocolboy) [reply src/LINEBot/Constant/ActionType.php]: @Vaduz You already done 😂

Comment (vocolboy) [reply src/LINEBot/Constant/ActionType.php]: It's work well but must modify composer.json to run test environment
```
{
  "license": "Apache License, Version 2.0",
  "authors": [
    {
      "name": "moznion",
      "email": "moznion@gmail.com"
    },
    {
      "name": "Satoru Yoshihara",
      "email": "vaduz0@gmail.com"
    }
  ],
  "require": {
    "php": ">=5.6",
    "slim/slim": "^3.5.0",
    "monolog/monolog": "^1.21.0",
    "linecorp/line-bot-sdk": "dev-master"
  },
  "autoload": {
    "psr-4": {
      "LINE\\": "src/"
    }
  },
  "repositories": [
    {
      "type": "vcs",
      "url": "https://github.com/vocolboy/line-bot-sdk-php"
    }
  ]
}
```

**Pull request merged**